test/e2e/suite_test.go

```diff
@@ -33,15 +38,42 @@ var (
 	ctx       context.Context
 )
 
+const (
+	Timeout  = time.Second * 30
+	Interval = time.Millisecond * 250
+)
+
 func TestAPIs(t *testing.T) {
 	gomega.RegisterFailHandler(ginkgo.Fail)
 	ginkgo.RunSpecs(t,
 		"End To End Suite",
 	)
 }
 
+func CreateClientUsingCluster() client.Client {
+	cfg := config.GetConfigOrDie()
+	gomega.ExpectWithOffset(1, cfg).NotTo(gomega.BeNil())
+
+	err := kueue.AddToScheme(scheme.Scheme)
+	gomega.ExpectWithOffset(1, err).NotTo(gomega.HaveOccurred())
+
+	// +kubebuilder:scaffold:scheme
+	client, err := client.New(cfg, client.Options{Scheme: scheme.Scheme})
+	gomega.ExpectWithOffset(1, err).NotTo(gomega.HaveOccurred())
+	return client
+}
+
+func KueueReadyForTesting(client client.Client) {
+	// To verify that webhooks are ready, let's create a simple resourceflavor
+	resourceKueue := kueuetest.MakeResourceFlavor("default").Obj()
+	gomega.Eventually(func() error {
+		return client.Create(context.Background(), resourceKueue)
+	}, Timeout, Interval).Should(gomega.Succeed())
+	util.ExpectResourceFlavorToBeDeleted(ctx, k8sClient, resourceKueue, true)
```

Comment:
Only now I understood that the lack of this was the cause for the failures :sweat_smile: 

Although, I would be happy to leave the ResourceFlavor created for the suite, instead of creating it in each test. Perhaps with a changed named: `SetupDefaultResourceFlavor`.

Again, feel free to leave it as a follow up.

Reply:
I'll kick off a few more tests and see if its stable.  If so, then I think we can merge.

Reply:
And then you could mark the e2e tests as mandatory in test-infra :)